src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/Umbraco.Cms.Integrations.Crm.ActiveCampaign.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<PackageId>Umbraco.Cms.Integrations.Crm.ActiveCampaign</PackageId>
+		<Title>Umbraco CMS Integrations: CRM - ActiveCampaign</Title>
+		<Description>An extension for Umbraco CMS providing a form picker and rendering component for ActiveCampaign.</Description>
+		<PackageIconUrl></PackageIconUrl>
+		<PackageProjectUrl>https://github.com/umbraco/Umbraco.Cms.Integrations</PackageProjectUrl>
+		<RepositoryUrl>https://github.com/umbraco/Umbraco.Cms.Integrations</RepositoryUrl>
+		<Version>1.0.0</Version>
+		<Authors>Umbraco HQ</Authors>
+		<Company>Umbraco</Company>
+		<ApplicationManifest>App_Plugins\UmbracoCms.Integrations\Crm\ActiveCampaign\package.manifest</ApplicationManifest>
```

Comment:
Do we need this here?  I've not seen it before so just flagging in case you are mixing two concepts here, Umbraco's package manifests and a Windows executables one.